jest.setup.js

```diff
@@ -0,0 +1 @@
+jest.setTimeout(30000)
```

Comment:
That is a very large timeout, does it have to be 30 seconds?

Reply:
Due to alert transitions it sometimes runs into a jest timeout(in some cases) which is by default 5s, 30s will ensure that transitions have enough time to complete, we can go for a lesser one too, like 10s. wdyt?